plugins/kubernetes/app/models/concerns/kubernetes/has_status.rb

```diff
@@ -0,0 +1,13 @@
+module Kubernetes
+  module HasStatus
+    STATUSES = %w[created spinning_up live spinning_down dead failed]
+
+    STATUSES.each do |s|
+      define_method("#{s}?") { status == s }
+    end
+  end
+
+  def status=(new_status)
+    super new_status.to_s
+  end
+end
```

Comment:
Just FYI: left the `fail!` method on the Release and `ReleaseDoc` classes because the implementation differs now.